tests/test_sundpaabudget.py

```diff
@@ -0,0 +1,97 @@
+from recipe_scrapers.sundpaabudget import SundPaaBudget
+from tests import ScraperTest
+
+
+class TestSundPaaBudgetScraper(ScraperTest):
+
+    scraper_class = SundPaaBudget
+
+    def test_host(self):
+        self.assertEqual("sundpaabudget.dk", self.harvester_class.host())
+
+    def test_title(self):
+        self.assertEqual(
+            "One pot pasta med kyllingekebab",
+            self.harvester_class.title(),
+        )
+
+    def test_category(self):
+        self.assertEqual(
+            "Kylling,One pot",
+            self.harvester_class.category(),
+        )
+
+    def test_total_time(self):
+        self.assertEqual(30, self.harvester_class.total_time())
+
+    def test_cook_time(self):
+        self.assertEqual(20, self.harvester_class.cook_time())
+
+    def test_prep_time(self):
+        self.assertEqual(10, self.harvester_class.prep_time())
+
+    def test_yields(self):
+        self.assertEqual("4 serving(s)", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://sundpaabudget.dk/wp-content/uploads/2021/08/20210803123311_IMG_0714.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_nutrients(self):
+        self.assertEqual(
+            {},
```

Comment:
Does this website have nutritional information available for other recipes?  (perhaps we should omit the `nutrients` implementation if we don't expect this scraper to find relevant content)

Reply:
I've looked through a few other recipes, especially ones branded as healthy, they contained no nutritional information either.

I figured that as long as it didn't blow up, I'd just include it to have support in case it was needed anywhere, but I agree that it should be removed if there's any possible of undesired side effects, since there doesn't seem to be any actual gain from having it.